libr/anal/p/anal_x86_cs.c

```diff
@@ -278,7 +278,7 @@ static char *getarg(struct Getarg* gop, int n, int set, char *setop, int sel, ut
 	case X86_OP_MEM:
 		{
 		// address = (base + (index * scale) + offset)
-		char buf_[BUF_SZ] = {0};
+		char buf_[BUF_SZ+32] = {0};
```

Comment:
```suggestion
		char buf_[BUF_SZ + 32] = {0};
```

Reply:
On second thought this `+32` is dumb anyways because the result gets `strncpy` into the smaller buffer again. Will try to check all `snprintf` return codes instead, hopefully without making the source very ugly

Reply:
Replaced using snprintf checks